src/tcgwars/logic/impl/gen3/TeamRocketReturns.groovy

```diff
@@ -485,7 +485,7 @@ public enum TeamRocketReturns implements LogicCardInfo {
                     self.damage = self.fullHP - hp(10)
                     bc "Buffer saved $self!"
                     prevent()
-                  }
+                  }, self.owner
```

Comment:
Does it work like this?

Reply:
Yes, it does.
```
● Glimwood Tangle is now in play.
● Inteleon VMAX used Hydro Snipe
● Hoppip resists WATER
● Inteleon VMAX did 30 damage to Hoppip
● Buffer coin flip result: HEADS
● Buffer saved Hoppip!
● Between turns
● Turn 4 begins
● Player B drew 1 card
● Between turns
● Turn 5 begins
● Player A drew 1 card
● Inteleon VMAX used Hydro Snipe
● Hoppip resists WATER
● Inteleon VMAX did 30 damage to Hoppip
● Buffer coin flip result: TAILS
● Hoppip was knocked out
● Player A took one prize card.
● Hoppip is now active
● Between turns
● Turn 6 begins
● Player B drew 1 card
● [DEV] A: added Paras (DAB 3) to HAND
● Between turns
● Turn 7 begins
● Player A drew 1 card
● Put Paras onto bench
● Between turns
● Turn 8 begins
● Player B drew 1 card
● Between turns
● Turn 9 begins
● Player A drew 1 card
● [DEV] A: added Parasect (DAB 4) to HAND
● Paras evolved into Parasect
● Parasect is now active
● [DEV] A: switched
● Parasect used Mushroom Tackle
● Coin flip result: TAILS
● Used Glimwood Tangle and discarded those flips
● Coin flip result: TAILS
● Parasect did 30 damage to Hoppip
● Buffer coin flip result: HEADS
● Buffer saved Hoppip!
● Between turns
● Turn 10 begins
● Player B drew 1 card
```
The coin flip works to save the Hoppip if heads, Glimwood Tangle will not activate for that effect. Glimwood Tangle does still work for the attacker though.